openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -157,8 +157,11 @@ def test_without_request_and_everyone_active_waffle(self):
         {'expected_count': 1, 'waffle_flag_metric_setting': [NAMESPACED_FLAG_NAME]},
         {'expected_count': 2, 'waffle_flag_metric_setting': [NAMESPACED_FLAG_NAME, NAMESPACED_FLAG_2_NAME]},
     )
-    @patch('openedx.core.djangoapps.waffle_utils.set_custom_metric')
-    def test_waffle_flag_metric_for_various_settings(self, data, mock_set_custom_metric):
+    @patch('openedx.core.djangoapps.waffle_utils.set_custom_attribute')
+    def test_waffle_flag_metric_for_various_settings(self, data, mock_set_custom_attribute):
```

Comment:
Test name?

Reply:
Handling in separate PR.  See other thread: https://github.com/edx/edx-platform/pull/25055#discussion_r495174789

Reply:
This one is already covered in https://github.com/edx/edx-platform/pull/25018 as it turns out.